README.md

```diff
@@ -409,6 +413,30 @@ for a single connection. The default values are shown below:
 ],
 ```
 
+The PhpRedis client supports extra options. The default values are shown below:
+
+```php
+'options' => [
+    ...
+
+    // The default number of attempts to retry the connection if the inititial
+    // connection has failed. A value of 0 instructs the
+    // client to throw an exception after the first failed attempt, while a
+    // value of -1 causes the client to continue to retry commands indefinitely.
+    'connector_retry_limit' => 20,
+
+    // The default amount of time (in milliseconds) that the client waits before
+    // retrying the connection attempt.
+    'connector_retry_wait' => 1000,
```

Comment:
How do these configuration paramters play together with `retry_limit` and `retry_wait`? Would it be possible to allow the `PhpRedisConnection` to reuse the connector logic (and retry configuration) of `PhpRedisConnector`?

Reply:
These settings allow for a different configuration on first connection. So the normal `retry_limit` and `retry_wait` are used for reconnecting after a failure. The `connector_` options are used when creating the connection for the first time.  

I've combined some of the retry logic, but I can't seem to find a way to simplify this. Maybe you have some ideas about it?

Reply:
I think the settings are fine, just a bit hard to understand in combination with the other ones (which have quite similar names too). Explaining their difference from the similar named settings is probably all you can do at this point.